### DISCUSSION TITLE
How to create a Thread or 'progressive prompt'?

### DISCUSSION THREAD

**Discussion by davidgwright** [betalgo/openai#89] [Q&A]
In Chat-GPT you can have multiple "threads" or "chats" which accumulate 'knowledge' and can be extended upon.

Is this a 'fine-tune' or is it an upcoming feature? That is, can the openai SDK here support a similar implementation?

Note:
- There's a token limit as well.
- A completion response contains an Id

**Comment by gotmike** [comment]
The way they suggest you handle this is to continue summarizing the conversation by feeding the convo back into the models and asking them to re-summarize then respond to a prompt, which is usually the last one or two lines of the convo.

Thats how u get around the token limit.

**Comment by kayhantolga** [comment]
I agree with @gotmike, but it will get quite expensive very quickly. 
You need to wait get access to your fine-tuned model, sometimes more than an hour. If you want to build conversation bot like chatgpt I don't think fine tune is a solution for you. But if you are looking for uploading an article, a book or a knowledge and ask questions about it, fine-tune is a pretty good solution.
I think they will introduce new way to use API for chat GPT model,

**Comment by davidgwright** [comment]
Would anyone say that this can now be done via the new ChatGPT support?
